frontend/dialogs/OBSBasicFilters.hpp

```diff
@@ -34,12 +34,7 @@ class OBSBasicFilters : public QDialog {
 	OBSSource source;
 	OBSPropertiesView *view = nullptr;
 
-	OBSSignal addSignal;
-	OBSSignal removeSignal;
-	OBSSignal reorderSignal;
-
-	OBSSignal removeSourceSignal;
-	OBSSignal renameSourceSignal;
+	std::vector<OBSSignal> sigs;
```

Comment:
```suggestion
	std::vector<OBSSignal> signals;
```

Reply:
Can't use `signals` as it's a Qt macro name.